packages/edit-widgets/src/components/widget-areas-block-editor-provider/index.js

```diff
@@ -54,7 +56,7 @@ export default function WidgetAreasBlockEditorProvider( {
 			widgets: select( editWidgetsStore ).getWidgets(),
 			reusableBlocks: ALLOW_REUSABLE_BLOCKS
 				? getEntityRecords( 'postType', 'wp_block' )
-				: [],
+				: EMPTY_ARRAY,
```

Comment:
The `ALLOW_REUSABLE_BLOCKS` flag is currently false for this editor screen, and we need a fallback with a stable reference.